app/serializers/curator/filestreams/file_set_serializer.rb

```diff
@@ -3,7 +3,20 @@
 module Curator
   class Filestreams::FileSetSerializer < CuratorSerializer
     schema_as_json do
-      attributes :ark_id
+      attributes :file_name_base, :position
```

Comment:
missing `schema_as_json root: :file_set`

Reply:
Actually that is left blank as the root is set in the inherited class. If you look at [app/serializers/curator_serializer.rb](app/serializers/curator_serializer.rb) it does not declare the root but every class that inherits from it does.